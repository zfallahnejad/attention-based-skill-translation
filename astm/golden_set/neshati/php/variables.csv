476,variables
250259,variables
1491895,variables
187606,variables
541091,variables
142162,variables
42139,variables
862594,variables
53114,variables
106224,variables
50079,variables
31671,variables
138475,variables
1415724,variables
454533,variables
298479,variables
218196,variables
1438393,variables
251311,variables
889949,variables
324584,variables
294972,variables
206403,variables
4833,variables
421223,variables
1230836,variables
227665,variables
2844319,variables
1679187,variables
229044,variables
496223,variables
661872,variables
215966,variables
