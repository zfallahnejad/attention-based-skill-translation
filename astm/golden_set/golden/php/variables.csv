476,47,54,0.46534653465346537
250259,37,33,0.5285714285714286
1491895,26,22,0.5416666666666666
187606,22,24,0.4782608695652174
541091,19,9,0.6785714285714286
42139,17,20,0.4594594594594595
142162,17,11,0.6071428571428571
862594,14,6,0.7
53114,13,13,0.5
31671,13,16,0.4482758620689655
106224,13,1,0.9285714285714286
50079,13,18,0.41935483870967744
1415724,12,8,0.6
138475,12,14,0.46153846153846156
218196,11,10,0.5238095238095238
298479,11,11,0.5
454533,11,14,0.44
1438393,11,13,0.4583333333333333
251311,10,11,0.47619047619047616
889949,10,9,0.5263157894736842
4833,9,8,0.5294117647058824
206403,9,13,0.4090909090909091
324584,9,13,0.4090909090909091
294972,9,12,0.42857142857142855
1679187,8,10,0.4444444444444444
421223,8,6,0.5714285714285714
227665,8,9,0.47058823529411764
2844319,8,9,0.47058823529411764
1230836,8,7,0.5333333333333333
496223,7,4,0.6363636363636364
215966,7,7,0.5
661872,7,8,0.4666666666666667
229044,7,9,0.4375
